packages/evmcrispr-terminal/src/utils/local-storage.ts

```diff
@@ -0,0 +1,12 @@
+import type { type Script } from '../components/library-scripts';
```

Comment:
The Vercel error

![image](https://user-images.githubusercontent.com/691510/209215108-d5c39e54-a972-408f-9015-86b72d173941.png)

Reply:
thanks for the help! @kamikazebr I think its working now :)